### DISCUSSION TITLE
Blazor WebAssembly without database

### DISCUSSION THREAD

**Discussion by pclaus1982** [abpframework/abp#21757] [Q&A]
Hi everyone!

I want to develop an application with Blazor that consumes a proxy for a service built with ABP. The idea is for this application to consume that proxy. I don't need a database.

Is there a way to create a web assembly project with Blazor using the CLI that sets up something like this?

Thank you in advance!

**Comment by maliming** [comment]
hi

You can select the `Tiered` option when creating the Blazor project.

The Blazor will be a pure UI and consume a remote API project.
